dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/VisitorLitmusC.java

```diff
@@ -539,8 +539,8 @@ public IExpr visitVarName(LitmusCParser.VarNameContext ctx){
 
     private IExpr getAddress(LitmusCParser.ReContext ctx){
         Expression address = (Expression)ctx.accept(this);
-        if(address instanceof IExpr){
-           return (IExpr)address;
+        if(address instanceof IExpr add){
+           return add;
```

Comment:
Use `addr` as abbreviation. I dislike `add` because it conflicts with "addition": if you just read `IExpr add` without context, it looks like it refers to a binary addition.
In general, I think we should aim to name variables in way that minimizes the amount of context needed to understand their meaning.